### PR TITLE
Add Ad Blocking Detection to Settings "View" mode

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue:

- #6961

## Relevant technical choices

I noticed a small issue in the markup for the web stories that caused extra spacing, so I fixed that here as well 🙂 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/adsense/components/settings/SettingsView.js [hunk 1]
```diff
@@ -39,6 +40,7 @@ import {
 	getAutoAdsDisabledMessage,
 } from './utils';
 import AdBlockingRecoveryCTA from '../common/AdBlockingRecoveryCTA';
+import { Fragment } from 'react';
```
Comment (aaemnnosttv) [inline assets/js/modules/adsense/components/settings/SettingsView.js:43]: `@wordpress/element` 😉

File: assets/js/modules/adsense/components/settings/SettingsView.js [hunk 2]
```diff
@@ -152,26 +165,64 @@ export default function SettingsView() {
...
+						) }
+						{ adBlockingRecoverySetupStatus === 'tag-placed' && (
+							<Fragment>
+								<p className="googlesitekit-settings-module__meta-item-data">
+									<strong>
+										{ __(
+											'Ad blocking recovery tag is placed',
+											'google-site-kit'
+										) }
+									</strong>
```
Comment (aaemnnosttv) [inline assets/js/modules/adsense/components/settings/SettingsView.js:206]: Unfortunately I think we need to deviate from the design a bit here (it's already not consistent with what we have now which is the problem) as all other setting values are not bold – with the current design this looks the same as a heading.

image

Let's remove the bold for consistency, and note this difference for QA

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
Thanks @tofumatt – this looks great, just a few small things.

File: assets/js/modules/adsense/components/settings/SettingsView.js (hunk 1 above)
Comment (tofumatt) [reply assets/js/modules/adsense/components/settings/SettingsView.js:43]: Argh, I blame VSCode! Thanks 😅

File: assets/js/modules/adsense/components/settings/SettingsView.js (hunk 2 above)
Comment (tofumatt) [reply assets/js/modules/adsense/components/settings/SettingsView.js:206]: Cool, will do 👍🏻

**Review by aaemnnosttv — APPROVED** [review]
LGTM, thanks @tofumatt !

**Pull request merged**